assets/i18n/en.json

```diff
@@ -750,6 +750,10 @@
     "I18N_LEARNER_DASHBOARD_ALL": "All",
     "I18N_LEARNER_DASHBOARD_ALL_GOALS_SECTION": "Edit Goals",
     "I18N_LEARNER_DASHBOARD_BRONZE_BADGE": "Bronze",
+    "I18N_LEARNER_DASHBOARD_CARD_BUTTON_REDO": "Redo",
```

Comment:
This needs translations, will you add them before rolling out the redesign?

Reply:
Sorry, I'm a bit confused. Am I translating these words into other languages?

Reply:
Just to clarify -- no need to add translations. Ours are supplied by translatewiki, so the only files that developers need to edit are en.json and qqq.json. Translatewiki will handle the rest.

Reply:
Thanks and sorry for the confusion